docs/developer/discounts/manual-discounts.mdx

```diff
@@ -355,3 +355,10 @@ The rest of the discount, **$13** was applied to the subtotal and next propagate
 its price was discounted by $10 (100 / 130 * 13).
 
 For more complex cases, please check [Examples](developer/discounts/examples.mdx) section.
+
+
+### Manual discounts with vouchers
+
+Manual discounts take precedence over vouchers, overriding them when applied. However, this does not entirely dissociate the voucher from the order.
+For example, if the manual discount is removed, the system reevaluates the voucher code associated with the order.
```

Comment:
Wouldn't be good to be more explicit here? 
What does `reevaluates` mean here? Does it mean, that the same voucher amount will be assigned back to the order? Or will the voucher amount be calculated based on the current order (like staff user modified lines/shipping method which changed the prices)? 
Does the `reevaluation` use de-normalized values assigned to the order? Will the flow be the same when voucher instance exists/was deleted in the meantime?